app/controllers/journeys/expenses/goodsToSellOrUse/GoodsToSellOrUseCYAController.scala

```diff
@@ -61,16 +63,13 @@ class GoodsToSellOrUseCYAController @Inject() (override val messagesApi: Message
 
   def onSubmit(taxYear: TaxYear, businessId: BusinessId): Action[AnyContent] = (identify andThen getData andThen requireData).async {
     implicit request =>
-      val journeyAnswers = (request.userAnswers.data \ businessId.value).as[GoodsToSellOrUseJourneyAnswers]
-      val context        = SubmissionContext(taxYear, Nino(request.user.nino), businessId, Mtditid(request.user.mtditid), ExpensesGoodsToSellOrUse)
+      val nextRoute = navigator.nextNormalRoute(GoodsToSellOrUseCYAPage, request.userAnswers, taxYear, businessId, accountingType = None).url
+      val result = service
+        .submitAnswers[GoodsToSellOrUseJourneyAnswers](taxYear, businessId, request.mtditid, ExpensesGoodsToSellOrUse, request.userAnswers)
```

Comment:
Can we use `SubmissionContext` for param holder?

Reply:
done